pkg/catalog/catalog.go

```diff
@@ -597,24 +596,23 @@ func (c *Catalog) GetTag(ctx context.Context, repository string, tagID string) (
 // the entry with ExpiredError if it has expired from underlying storage.
 func (c *Catalog) GetEntry(ctx context.Context, repository string, reference string, path string, _ GetEntryParams) (*DBEntry, error) {
 	repositoryID := graveler.RepositoryID(repository)
-	ref := graveler.Ref(reference)
-	p := Path(path)
+	getRef := graveler.Ref(reference)
```

Comment:
```suggestion
	reference := graveler.Ref(reference)
```

Reply:
The 'reference' here is already a `string` type